src/Nethermind/Nethermind.Consensus/Processing/ProcessingStats.cs

```diff
@@ -74,33 +87,71 @@ public void UpdateStats(Block? block, IBlockTree blockTreeCtx, int recoveryQueue
+                    decimal totalTxPerSecond = totalMicroseconds == 0 ? -1 : Metrics.Transactions / totalMicroseconds * 1000 * 1000;
+                    decimal totalBlocksPerSecond = totalMicroseconds == 0 ? -1 : _totalBlocks / totalMicroseconds * 1000 * 1000;
+                    decimal txps = chunkMicroseconds == 0 ? -1 : chunkTx / chunkMicroseconds * 1000 * 1000;
+                    decimal bps = chunkMicroseconds == 0 ? -1 : chunkBlocks / chunkMicroseconds * 1000 * 1000;
+                    decimal chunkMs = (chunkMicroseconds == 0 ? -1 : chunkMicroseconds / 1000);
+                    decimal runMs = (runMicroseconds == 0 ? -1 : runMicroseconds / 1000);
+                    string blockGas = Evm.Metrics.BlockMinGasPrice != decimal.MaxValue ? $" Gas gwei: {Evm.Metrics.BlockMinGasPrice:N2} .. {whiteText}{Math.Max(Evm.Metrics.BlockMinGasPrice, Evm.Metrics.BlockEstMedianGasPrice):N2}{resetColor} ({Evm.Metrics.BlockAveGasPrice:N2}) .. {Evm.Metrics.BlockMaxGasPrice:N2}" : "";
+                    if (chunkBlocks > 1)
+                    {
+                        _logger.Info($"Processed   {block.Number - chunkBlocks + 1,9}...{block.Number,9} | {chunkMs,9:N2} ms  | slot {runMs,7:N0} ms     |{blockGas}");
+                    }
+                    else
+                    {
+                        var chunkColor = chunkMs switch
+                        {
+                            < 200 => greenText,
+                            < 400 => whiteText,
+                            < 700 => yellowText,
+                            < 1000 => orangeText,
+                            _ => redText
+                        };
+                        _logger.Info($"Processed     {block.Number,9}           | {chunkColor}{chunkMs,9:N2}{resetColor} ms  | slot {runMs,7:N0} ms     |{blockGas}");
+                    }
+
+                    var mgasColor = chunkMGas switch
+                    {
+                        > 25 => greenText,
+                        > 15 => whiteText,
+                        > 10 => yellowText,
+                        > 6 => orangeText,
+                        _ => redText
+                    };
+                    _logger.Info($"- Block{(chunkBlocks > 1 ? "s" : " ")}{(chunkBlocks == 1 ? mgasColor : "")}           {chunkMGas,7:F2}{resetColor} MGas   | {chunkTx,6:N0}    txs | calls {chunkCalls,6:N0} {greyText}({chunkEmptyCalls,3:N0}){resetColor}  | sload {chunkSload,7:N0} | sstore {chunkSstore,6:N0} | create {chunkCreates,3:N0}{(currentSelfDestructs - _lastSelfDestructs > 0 ? $"{greyText}({-(currentSelfDestructs - _lastSelfDestructs),3:N0}){resetColor}" : "")}");
+                    _logger.Info($"- Block Throughput {mgasPerSecond,7:F2} MGas/s | {txps,9:F2} t/s |         {bps,7:F2} b/s | recv  {recoveryQueueSize,7:N0} | proc   {blockQueueSize,6:N0}");
+                    // Only output the total throughput in debug mode
```

Comment:
Do we want multiline report here?

Reply:
Multi line breaks into the date column and ruins the formatting; is ok for occasional ones but these are every 5 or 12 secs (depending on chain)